src/reducer/videoReducer.js

```diff
@@ -0,0 +1,87 @@
+import { errorToast, successToast } from '../components/Toast/Toast'
+import {
+  ADD_TO_PLAYLIST,
+  CREATE_PLAYLIST,
+  DATA_FROM_SERVER,
+  ADD_TO_HISTORY,
+  LIKED_VIDEOS,
+  SEARCH_VIDEO,
+  WATCH_LATER
+} from '../constants/reducerConstants'
+import { checkItem } from '../utils/checkItem'
+
+export default function videoReducer(state, { type, payload }) {
+  switch (type) {
+    case DATA_FROM_SERVER:
+      return {
+        ...state,
+        videos: payload
+      }
+    case LIKED_VIDEOS:
+      state.likedVideos.includes(payload)
```

Comment:
Nice use of Array methods like includes etc.

Reply:
:)